### ISSUE TITLE
Can this software be run without protein IDs?

### ISSUE THREAD

**Issue by BioMatt** [kiharalab/GO2Sum#3]
I have a list of GO annotations created with [FANTASIA](https://github.com/MetazoaPhylogenomicsLab/FANTASIA). Therefore, I do not have protein ID values for many genes. However, this software seems potentially very useful for summarizing active pathways with this GO information. Is there any way to run it without protein IDs in the input file? Thank you.

**Comment by SwagarikaGiri** [comment]
Yes. It can run without Protein IDs. You can name it Protein1, Protein2, and so on instead of real protein names.
